draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3634,6 +3634,21 @@ Upgrade: websocket
    (<xref target="attack.protocol.element.length"/>).
 </t>
 <t>
+   Because it is used for message delimitation in HTTP/1.1, the Content-Length
+   header field value can affect the security of the protocol. This is true even
+   when the current connection is not using HTTP/1.1, because HTTP is an
+   intermediated protocol, and messages might be transmitted using other versions
+   protocol.
+</t>
+<t>
+   As a result, implementations &MUST-NOT; forward a message with a
+   Content-Length header field value that is known to be incorrect, unless
+   its value is being used for message delimitation. Specific versions of HTTP
```

Comment:
I don't get the "unless" clause. Should it say *not* used for message delimitation?

Reply:
right, both that and the ending sentence are unnecessary.